src/string_util.cc

```diff
@@ -32,7 +38,8 @@ static const int64_t kUnitsSize = arraysize(kBigSIUnits);
 
 void ToExponentAndMantissa(double val, double thresh, int precision,
                            double one_k, std::string* mantissa,
-                           int64_t* exponent) {
+                           int64_t* exponent,
+                           bool inclusiveBigThreshhold = false) {
```

Comment:
seems like we only set this to true when we're doing base-10.  it might be worth having a comment why this is necessary for base-10.  or even better, renaming the variable to indicate that directly.